halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/telemetry/TelemetryEditCommand.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.cli.command.v1.config.telemetry;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.netflix.spinnaker.halyard.cli.command.v1.config.AbstractConfigCommand;
+import com.netflix.spinnaker.halyard.cli.services.v1.Daemon;
+import com.netflix.spinnaker.halyard.cli.services.v1.OperationHandler;
+import com.netflix.spinnaker.halyard.config.model.v1.node.Telemetry;
+import lombok.AccessLevel;
+import lombok.Getter;
+
+@Parameters(separators = "=")
+public class TelemetryEditCommand extends AbstractConfigCommand {
+  @Getter(AccessLevel.PUBLIC)
+  private String commandName = "edit";
+
+  @Getter(AccessLevel.PUBLIC)
+  private String shortDescription = "Edit Spinnaker's telemetry settings.";
+
+  @Parameter(names = "--endpoint", description = "Set the endpoint for telemetry metrics.")
+  private String endpoint;
+
+  @Override
+  protected void executeThis() {
+    String currentDeployment = getCurrentDeployment();
+    Telemetry telemetry =
+        new OperationHandler<Telemetry>()
+            .setOperation(Daemon.getTelemetry(currentDeployment, false))
+            .setFailureMesssage("Failed to load telemetry settings.")
+            .get();
+
+    telemetry.setEndpoint(isSet(endpoint) ? endpoint : telemetry.getEndpoint());
```

Comment:
Might be more clear as:
```
if (isSet(endpoint)) {
  telemetry.setEndpoint(endpoint);
}
```
As it, it's a bit confusing that the `false` branch of the ternary is just the no-op `telemetry.setEndpoint(telemetry.getEndpoint());`

Reply:
I've been changing this everywhere I'm already touching and trying to write new code more clearly.  Unfortunately 90% of halyard is people copy-pasting existing code, so if we don't make an effort to make it better it will only get worse.

I'm not overly concerned about this particular pattern, but I *am* concerned about allowing more and more unclear code in Halyard because there's already so much unclear code, as that will get harder and harder to maintain.

Reply:
@ezimanyi I've addressed your feedback here. :)